[2023-12-16T18:07:06.328+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.extract_gps manual__2023-12-16T18:07:01.673409+00:00 [queued]>
[2023-12-16T18:07:06.335+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.extract_gps manual__2023-12-16T18:07:01.673409+00:00 [queued]>
[2023-12-16T18:07:06.336+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-16T18:07:06.348+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_gps> on 2023-12-16 18:07:01.673409+00:00
[2023-12-16T18:07:06.356+0000] {standard_task_runner.py:57} INFO - Started process 11346 to run task
[2023-12-16T18:07:06.358+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_data_pipeline', 'extract_gps', 'manual__2023-12-16T18:07:01.673409+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/taxi_etl.py', '--cfg-path', '/tmp/tmparoncf3a']
[2023-12-16T18:07:06.360+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask extract_gps
[2023-12-16T18:07:06.402+0000] {task_command.py:416} INFO - Running <TaskInstance: NYC_Taxi_data_pipeline.extract_gps manual__2023-12-16T18:07:01.673409+00:00 [running]> on host 6abb98cbcbb2
[2023-12-16T18:07:06.467+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='islam' AIRFLOW_CTX_DAG_ID='NYC_Taxi_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_gps' AIRFLOW_CTX_EXECUTION_DATE='2023-12-16T18:07:01.673409+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-16T18:07:01.673409+00:00'
[2023-12-16T18:07:06.615+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-16T18:07:06.624+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_data_pipeline, task_id=extract_gps, execution_date=20231216T180701, start_date=20231216T180706, end_date=20231216T180706
[2023-12-16T18:07:06.652+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-16T18:07:06.672+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
